app/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "network": "development",
+    "testnet": {
+        "services": "0x60b1a811f5ea71806bb3bc0e5b74920eeae91913",
+        "endpoint": "http://?.?.?.?:8545"
```

Comment:
Since the truffle.js relies on hostnames following the scheme in gnidan/docker-eth-dev, should the dapp part be set up that way as well?